app/src/org/commcare/dalvik/activities/LoginActivity.java

```diff
@@ -571,12 +556,21 @@ public CustomProgressDialog generateProgressDialog(int taskId) {
         }
         return dialog;
     }
-    
+
     @Override
     public boolean isBackEnabled() {
-        return false;
+        return true;
     }
-    
+
+    @Override
+    public void onBackPressed() {
+        super.onBackPressed();

```

Comment:
This is the behavior for the "Back" button on the action bar, which isn't what we want, right? I don't think it makes sense for that to ever take you out of CommCare.